### ISSUE TITLE
Labels of 3d semseg

### ISSUE THREAD

**Issue by mc171819** [waymo-research/waymo-open-dataset#579]
Hi, when I following the tutorial of 3d semseg, I found that it seems that only a few frames in a sequence have 3d semseg, when every frame has a 3d bbox groundtruth, is that correct?

**Comment by alexzzhu** [comment]
Yes that's correct, the semseg labels are only provided for a subset of the frames.
